experiments/run_slicegpt.py

```diff
@@ -8,9 +8,13 @@
 import shutil
 
 import torch
-import wandb
+from transformers.models.llama.modeling_llama import LlamaConfig
```

Comment:
I think we might be missing one abstraction. We shouldn't need any model specific imports here. `hf_utils.get_model_and_tokenizer` abstracts the model type away; we want the same for saving HF-compatible sliced models. This `save` abstraction will also avoid the `Sliced<Model>ForCausalLM` imports here.